plugins/generator-1.16.5/forge-1.16.5/templates/json/block_states.json.ftl

```diff
@@ -1,4 +1,63 @@
 <#if data.rotationMode?? && (data.rotationMode == 1 || data.rotationMode == 3)>
+<#if data.enablePitch>
+{
+  "variants": {
+    "face=floor,facing=north": {
+      "model": "${modid}:block/${registryname}",
+      "x": 270

```

Comment:
The model rotations here don't match that of levers (floor, north should be the default state, with no rotations)

Reply:
The way I thought: the default AttachFace value for custom blocks is WALL (and with the pitch disabled the Block normally "looks at" wall), so the default (no-rotation) state should be the one targeting wall on the north.

Reply:
Buttons, lever and grindstone all use "north,floor" as the state with no rotation. I think it'd be better to follow the same convention

Reply:
But should I change default property value too?

Reply:
You can keep it "wall"

Reply:
:heavy_check_mark: Fixed